milestones/3_human_brain_mapping_journal_submission/experiments/ex_1_pli_generation_comparison/README.md

```diff
@@ -52,6 +53,17 @@ This version yield the following result:
 | B/U  | 85% | 69% |
 | B/PR | 68% | 68% |
 
+## Feature generation from old graph
+see: `generate_features_from_old_graph.m`
+
+In this analysis we tried to reassure ourselves that everything in the machine learning pipeline was working properly by generating the features.csv matrix from the matrix that Jason generated. We were able to re-generate the exact number we've outlined in the paper. This means that the problem lies in the generation of the (w)PLI graphs.
+
+It yield:
+
+|      | AEC | PLI |
+|------|-----|-----|
+| B/U  | 85% | 78% |
+| B/PR | 68% | 78% |
 
 ## Lucrezia Liuzzi PLI (no surrogate)
 see: `generate_liuzzi_pli.m`
```

Comment:
If possible do you think you can add short definitions for each variable (% what it is) for the lines missing, if you have that info that would help every step of the code. 
   (ht = hilbert(Vfilt);
    ht = ht(cut+1:end-cut,:);
    ht = bsxfun(@minus,ht,mean(ht,1));
and N for bandwdith